Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.11
+
+RUN mkdir /incident_scraper
+
+COPY /incident_scraper /incident_scraper
+COPY pyproject.toml /incident_scraper
+
+WORKDIR /incident_scraper
+ENV PYTHONPATH=${PYTHONPATH}:${PWD}
+
+RUN pip3 install poetry
+RUN poetry config virtualenvs.create false
+RUN poetry install --no-dev
```

Docker images work off of [layers](https://docs.docker.com/build/guide/layers/), which mean the state of an image is captured at every individual command. It can be useful to reduce image size by combining multiple `RUN` steps into a single step. This can increase build time (if, for instance, one aspect of that `RUN` is unaffected), but it can reduce final image size significantly. Depending on which you'd like to optimize (build time vs image size), I might recommend combining these into a single step:

```suggestion
RUN pip3 install poetry && \
    poetry config virtualenvs.create false && \
    poetry install --no-dev
```